app/screens/channel/channel_title/index.tsx

```diff
@@ -0,0 +1,266 @@
+                name='chevron-down'
+            />
+        );
+    };
+
+    return (
+        <TouchableOpacity
+            testID={'channel.title.button'}
+            style={style.container}
+            onPress={onPress}
+        >
+            <View style={[style.wrapper, {width: `${wrapperWidth}%`}]}>
+                {renderArchiveIcon()}
+                <Text
+                    ellipsizeMode='tail'
+                    numberOfLines={1}
+                    style={style.text}
+                    testID='channel.nav_bar.title'
+                >
+                    {renderChannelDisplayName()}
+                </Text>
+                {renderChannelIcon()}
+                {renderIcon()}
+                {renderMutedIcon()}
+            </View>
+            {renderHasGuestsText()}
+        </TouchableOpacity>
+    );
+};
+
+const getStyle = makeStyleSheetFromTheme((theme) => {
+    return {
+        container: {
+            flex: 1,
+        },
+        wrapper: {
+            alignItems: 'center',
+            flex: 1,
+            position: 'relative',
+            top: -1,
+            flexDirection: 'row',
+            justifyContent: 'flex-start',
+        },
+        icon: {
+            color: theme.sidebarHeaderTextColor,
+            marginHorizontal: 1,
+        },
+        emoji: {
+            marginHorizontal: 5,
+        },
+        text: {
+            color: theme.sidebarHeaderTextColor,
+            fontSize: 18,
+            fontWeight: 'bold',
+            textAlign: 'center',
+            flex: 0,
+            flexShrink: 1,
+        },
+        channelIconContainer: {
+            marginLeft: 3,
+            marginRight: 0,
+        },
+        muted: {
+            marginTop: 1,
+            opacity: 0.6,
+            marginLeft: 0,
+        },
+        archiveIcon: {
+            fontSize: 17,
+            color: theme.sidebarHeaderTextColor,
+            paddingRight: 7,
+        },
+        guestsWrapper: {
+            alignItems: 'flex-start',
+            flex: 1,
+            position: 'relative',
+            top: -1,
+            width: '90%',
+        },
+        guestsText: {
+            color: theme.sidebarHeaderTextColor,
+            fontSize: 14,
+            opacity: 0.6,
+        },
+    };
+});
+
+type ChannelTitleInputProps = {
+    canHaveSubtitle: boolean;
+    channel: ChannelModel;
+    currentUserId: string;
+    teammateId?: string;
+    onPress: () => void;
+};
+
+type ChannelTitleProps = ChannelTitleInputProps & {
+    channelInfo: ChannelInfoModel[];
+    channelSettings: MyChannelSettingsModel[];
+    database: Database;
+    teammate?: UserModel;
+    teammateId: string;
+};
+
+const ChannelTitle: React.FunctionComponent<ChannelTitleInputProps> = withDatabase(
+    withObservables(['channel', 'teammateId'], ({channel, teammateId, database}: { channel: ChannelModel; teammateId: string; database: Database }) => {
+        return {
+            channelInfo: channel.info.observe(),
```

Comment:
try this
```suggestion
            channelInfo: channel.info.findAndObserve(channel.id),
```

or 

```suggestion
            channelInfo: database.get(MM_TABLES.SERVER.CHANNEL_INFO).findAndObserve(channel.id),
```

Reply:
Second method worked.  `findAndObserve` does not exist on type `Query` and `info` is a query.

Reply:
will you be making the change then ?